t/venom/test_suites/inline/l2/05_setup_packetfence.yml

```diff
@@ -0,0 +1,300 @@
+name: Enable Inline l2 configuration in PacketFence
+testcases:
+- name: get_login_token
+  steps:
+  - type: get_login_token
+
+- name: Configure interface inlinel2 as type inline l2
+  steps:
+  - type: http
+    method: PATCH
+    url: '{{.pfserver_webadmin_url}}/api/v1/config/interface/inlinel2'
+    ignore_verify_ssl: true
+    body: >-
+     {
+       "id": "inlinel2",
+       "isClone": false,
+       "isNew": false,
+       "prefixRouteName": "",
+       "additional_listening_daemons": [],
+       "coa": null,
+       "dhcpd_enabled": "enabled",
+       "dns": "8.8.8.8",
+       "high_availability": 0,
+       "hwaddr": "92:ed:ee:36:e9:6a",
+       "ifindex": "10",
+       "ipv6_address": null,
+       "ipv6_prefix": null,
+       "is_running": true,
+       "master": null,
+       "name": "inlinel2",
+       "nat_enabled": "enabled",
+       "network": null,
+       "network_iseditable": false,
+       "networks": [],
+       "not_editable": false,
+       "reg_network": null,
+       "split_network": null,
+       "tenant_id": 1,
+       "type": "inlinel2",
+       "vip": null,
+       "vlan": null,
+       "ipaddress": "192.168.2.1",
+       "netmask": "255.255.255.0"
```

Comment:
NetFlow option is missing

Reply:
In the case of inline l2 interface i am not sure it needs to be there. It´s more a network option.

Reply:
The option is available on the form when you edit an interface.
It is defined here, it's automatically enabled on the network.

Reply:
Let's keep like this for now.